### ISSUE TITLE
Troubleshooting

### ISSUE THREAD

**Issue by xfq** [atom/docs#48]
It'd be nice to have a troubleshooting section. For example, how to deal with errors in config/script/keymap/stylesheet/snippet files, [how to start a clean Atom instance](https://discuss.atom.io/t/how-to-start-a-clean-atom-instance/14656), what to do when/after Atom crashes, how to debug the problem (Atom is a hackable editor ;-), how to report a bug (link to the [bug-report](https://atom.io/packages/bug-report) package, perhaps), etc. A link to the [FAQ](https://discuss.atom.io/t/the-one-true-faq/14981) in Atom Discussion is helpful, too.

**Comment by mnquintana** [comment]
At least some of this info is in https://atom.io/docs/latest/hacking-atom-debugging, but it's probably missing something.

**Comment by izuzak** [comment]
@xfq Yeah, improving the debugging guide which @mnquintana linked to and the [instructions for reporting issues](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#submitting-issues) is on our radar. Thanks for opening an issue and for the specific suggestions. If you have any other suggestions -- let us know.
